panoramapublic/src/org/labkey/panoramapublic/PanoramaPublicController.java

```diff
@@ -590,6 +860,16 @@ private void validateAction(CopyExperimentForm form)
             {
                 throw new UnauthorizedException("You do not have permissions to copy this experiment.");
             }
+            _journalExperiment = JournalManager.getJournalExperiment(_experiment.getId(), _journal.getId());
+            if(_journalExperiment == null)
+            {
+                throw new NotFoundException("Could not find an entry in JournalExperiment table for experimentId " + _experiment.getId()
+                + " and journalId " + _journal.getId());
+            }
+            if(_journalExperiment.getCopied() != null)
+            {
+                throw new UnsupportedOperationException(String.format("The experiment ID %d has already been copied.  It cannot be copied again.", _experiment.getId()));
```

Comment:
This will get reported to the mothership exception reporting. Not a big deal, but this looks like a data problem, not a code bug. Consider using BadRequestException instead, or something else that implements SkipMothershipLogging